### PR TITLE
Educash_shortcode file.

### PR DESCRIPTION
**Pull request by imayankjoshi**

### CODE REVIEW

File: edugorilla.php [hunk 1]
```diff
@@ -46,12 +46,21 @@ function create_edugorilla_lead_table()
                                             comments varchar(500) DEFAULT 'No comment' NOT NULL,
                                             PRIMARY KEY  (id)
                                         ) $charset_collate;"; //Defining query to create table.
+$table_name4 = $wpdb->prefix . 'transaction_history';
```
Comment (rohitmanglik) [inline edugorilla.php]: put `educash_transaction_history`

File: shortcode_educash.php [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+     echo '<h4><b>Your educash is : '.$cash."</b></h4>";
+     }
+
+     else echo '<h4><b>Your educash is : '.$cash."</b></h4>";
+     ?></div><br><?php
+  }
+ }
+?>
+
+<style>
```
Comment (rohitmanglik) [inline shortcode_educash.php]: Put this in plugin wide stylesheet

File: shortcode_educash.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+   if(is_user_logged_in()){
+     $current_user = wp_get_current_user();
+     $name = $current_user->user_login;
+     $sql = "SELECT educash_added FROM wp_educash_deals WHERE client_name = '$name'";
+     $myrow = $wpdb->get_results($sql);
+     ?><br><br><div class="boxed"><?php
+     if(count($myrow)>0){
+
+     echo '<h3><b>Hello '.$current_user->user_login.',</b></h3>';
+     //echo '<h3><b>Hello mayank chandra bhuban singh joshi ji on this way,</b></h3>';
```
Comment (rohitmanglik) [inline shortcode_educash.php]: @shubhiD Any suggestions on Transaction history layout?

Comment (rohitmanglik) [inline shortcode_educash.php]: @anantharam your take?

File: edugorilla.php [hunk 4]
```diff
@@ -677,6 +686,6 @@ function vc_remove_wp_ver_css_js( $src ) {
 
 
 include_once plugin_dir_path(__FILE__) . "email_setting.php";
-
+include_once plugin_dir_path(__FILE__) . "shortcode_educash.php";
```
Comment (rohitmanglik) [inline edugorilla.php]: Create an "inc" folder and put this file over there.

Comment (imayankjoshi) [inline edugorilla.php]: I have to create a folder name "inc", that's fine I will do it now.

File: shortcode_educash.php (hunk 2 above)
Comment (imayankjoshi) [inline shortcode_educash.php]: By plugin-wide style sheet, you mean I have to put this styling part in other ( .css) file and include that file in css folder.

Comment (rohitmanglik) [inline shortcode_educash.php]: Yes there should be one css file for whole plugin. In this CSS file, before every code mention who wrote it and what functionality it supports.

However for third party libraries, we can include them separately.

**Pull request closed without merging**